cmd/osbuild-worker/jobimpl-osbuild.go

```diff
@@ -612,7 +612,9 @@ func (impl *OSBuildJobImpl) Run(job worker.Job) error {
 		artifact := jobTarget.OsbuildArtifact
 		switch targetOptions := jobTarget.Options.(type) {
 		case *target.WorkerServerTargetOptions:
-			targetResult = target.NewWorkerServerTargetResult(&artifact)
+			targetResult = target.NewWorkerServerTargetResult(&target.WorkerServerTargetResultOptions{
+				Filename: path.Join(jobTarget.OsbuildArtifact.ExportFilename),
```

Comment:
Nitpick: so this is not really a `filename`, but a path to the artifact relative to the job artifacts directory. I think that it would make sense to rename the `Filename` struct member to something that expresses this fact.